com.sap.cloud.lm.sl.cf.web/src/main/java/com/sap/cloud/lm/sl/cf/web/monitoring/FssMonitor.java

```diff
@@ -17,8 +17,8 @@
 @Component
 public class FssMonitor {
 
-    Map<File, Long> usedSpaceMap = new Hashtable<>(1);
```

Comment:
There's an important difference between `Hashtable` and `HashMap`. `Hashtable` is synchronized. The method `calculateUsedSpace` may get called by different threads, so use a synchronized map implementation in order to avoid problems.